lighthouse-core/audits/autocomplete.js

```diff
@@ -43,22 +194,80 @@ class AutocompleteAudit extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.FormInput} input
+   * @return {{hasValidTokens: Boolean, inValidOrder?: Boolean}}
```

Comment:
use `boolean` instead of `Boolean` 
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html